### PR TITLE
IA-754 org units

### PR DESCRIPTION
**Pull request by beygorghor**
- reordering and rename org units filters
- put location limit inside filters
-  new color picker 
- change of color if adding one new search
Screenshot 2021-09-29 at 13 54 34
Screenshot 2021-09-29 at 13 52 03

### CODE REVIEW

File: iaso/api/serializers.py [hunk 1]
```diff
@@ -143,6 +158,9 @@ class Meta:
             "sub_source",
             "org_unit_type_name",
             "parent",
+            "latitude",
+            "longitude",
+            "altitude",
```
Comment (olethanh) [inline iaso/api/serializers.py:163]: you have added these field but I don't see where they are used?

File: hat/assets/js/apps/Iaso/components/forms/ColorPicker.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        id: 'iaso.label.color',
+    },
+});
+
+const useStyles = makeStyles(theme => ({
+    button: {
+        border: `3px solid ${theme.palette.ligthGray.border}`,
+        borderRadius: theme.spacing(4),
+        width: theme.spacing(4),
+        height: theme.spacing(4),
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/components/forms/ColorPicker.js:27]: I find the button a bit too big, maybe `theme.spacing(2)` is enough?

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 3]
```diff
@@ -77,7 +79,7 @@
     "iaso.forms.dialog.deleteFormText": "This operation cannot be undone.",
     "iaso.forms.dialog.deleteFormTitle": "Are you sure you want to delete this form?",
     "iaso.forms.form_id": "Form id",
-    "iaso.forms.hasInstances": "Forms",
+    "iaso.forms.hasInstances": "Forms submissions",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:82]: I think it's "Form submissions"

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 4]
```diff
@@ -144,8 +147,8 @@
     "iaso.instance.period": "Period",
     "iaso.instance.selectedOrgUnit": "Selected Org unit",
     "iaso.instance.table.label.instanceHeaderTooltip": "Label: {label} - Name: {key}",
-    "iaso.instance.undelete": "Mark isntance(s) as not deleted",
     "iaso.instance.unDeleteCount": "Mark {count} isntance(s) as not deleted",
+    "iaso.instance.undelete": "Mark isntance(s) as not deleted",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:151]: "isntance(s)" 😂

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 5]
```diff
@@ -53,10 +55,10 @@
     "iaso.errors.label": "Une erreur est survenue.",
     "iaso.errors.notFound": "Page introuvable !",
     "iaso.exporStatus.createdAt": "Quand",
+    "iaso.exportRequest.Status": "Status",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:58]: "Statut"

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 6]
```diff
@@ -141,8 +144,8 @@
     "iaso.instance.period": "Période",
     "iaso.instance.selectedOrgUnit": "Unité d'organisation sélectionnée",
     "iaso.instance.table.label.instanceHeaderTooltip": "Label: {label} - Nom: {key}",
-    "iaso.instance.undelete": "Marquer une(des) soumission(s) comme non effacée(s)",
     "iaso.instance.unDeleteCount": "Marquer {count} soumission(s) comme non effacée(s)",
+    "iaso.instance.undelete": "Marquer une(des) soumission(s) comme non effacée(s)",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:148]: "non-effacée(s)"

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 7]
```diff
@@ -409,8 +417,8 @@
     "iaso.orgUnits.parentsMulti": "Parent(s)",
     "iaso.orgUnits.removeFromGroups": "Retirer du(des) groupe(s)",
     "iaso.orgUnits.search": "Rechercher une unité d'organisation",
-    "iaso.orgUnits.selectionAction": "Avec la sélection d'unité d'organisation",
     "iaso.orgUnits.selectOrgUnit": "Merci de sélectionner une unité d'organisation",
+    "iaso.orgUnits.selectionAction": "Avec la sélection d'unité d'organisation",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:421]: "unités d'organisation"

File: hat/assets/js/apps/Iaso/domains/forms/messages.js [hunk 8]
```diff
@@ -161,13 +161,17 @@ const MESSAGES = defineMessages({
...
+        defaultMessage: 'Validation status',
+    },
     showDeleted: {
         id: 'iaso.forms.showDeleted',
         defaultMessage: 'Show deleted',
     },
     hasInstances: {
         id: 'iaso.forms.hasInstances',
-        defaultMessage: 'Forms',
+        defaultMessage: 'Forms submissions',
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/forms/messages.js:174]: "Form submissions"

File: hat/assets/js/apps/Iaso/domains/instances/components/InstancesFiltersComponent.js [hunk 9]
```diff
@@ -198,13 +197,15 @@ const InstancesFiltersComponent = ({
...
                                 onEnterPressed={() => handleSearch()}
                             />
-                            <OrgUnitTreeviewModal
-                                toggleOnLabelClick={false}
-                                titleMessage={MESSAGES.search}
-                                onConfirm={orgUnitId => {
-                                    onSelectOrgUnitFromTree(orgUnitId);
-                                }}
-                            />
+                            <Box mt={1}>
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/instances/components/InstancesFiltersComponent.js:200]: This might break alignment at the next update of blsq-comp

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/instances/components/InstancesFiltersComponent.js:200]: It's my fault, I made then reverted a stupid change

File: hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgunitsMapComponent.js [hunk 10]
```diff
@@ -99,11 +98,15 @@ class OrgunitsMap extends Component {
     shouldComponentUpdate(nextProps) {
         return (
             !isEqual(nextProps.orgUnits, this.props.orgUnits) ||
-            !isEqual(getColorsFromParams(nextProps.params, this.props.params))
+            !isEqual(
+                getColorsFromParams(nextProps.params),
+                getColorsFromParams(this.props.params),
+            )
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgunitsMapComponent.js:104]: 😃👍

**Review by quang-le — CHANGES_REQUESTED** [review]
Maybe update bluesquare-components to the latest version to see if the treeview is still aligned.

Otherwise mostly nitpicking with translations

**Review by quang-le — APPROVED** [review]
I made the small changes from my comments.
Very nice changes.
I still think the color picker should not be on top, but I can live with it

**Pull request merged**